container/devel:openQA:ci/base/Dockerfile

```diff
@@ -4,7 +4,7 @@
 # in one go
 # see https://progress.opensuse.org/issues/67855
 #!NoSquash
-FROM opensuse/leap:15.4
+FROM opensuse/leap:15.5
```

Comment:
[poo#130597](https://progress.opensuse.org/issues/130597) is probably more suitable for this commit in particular. The overall idea with this and connected PRs is to fulfill [poo#130582](https://progress.opensuse.org/issues/130582).

I would think that staying up-to-date with the latest supported version of Leap is quite self-explanatory even without a specific ticket though.

Reply:
And of course, this is needed for https://github.com/os-autoinst/openQA/pull/5199 to pass 😄

Reply:
Self-explanatory yes. But we also have tickets to prioritize, and if you think it's worth looking into future tickets asap you could at least assign yourself to avoid people wondering why we have tickets for stuff that's already done... (and no, it's not done, but that's where it gets confusing)

Reply:
> you could at least assign yourself to avoid people wondering why we have tickets for stuff that's already done

Yes, that's important.